src/Microsoft.Data.SqlClient.sln

```diff
@@ -195,7 +195,6 @@ EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "src", "src", "{4F3CD363-B1E6-4D6D-9466-97D78A56BE45}"
 	ProjectSection(SolutionItems) = preProject
 		Directory.Build.props = Directory.Build.props
-		NuGet.config = NuGet.config
```

Comment:
Why did this get removed?

Reply:
I suppose this should actually go against main. The file hasn't existed for a while, and it was annoying me had a warning in the solution explorer.